doc/API.md

```diff
@@ -131,12 +131,20 @@ As an alternative, it is possible to create a Libp2p instance with the construct
 
 ```js
 const Libp2p = require('libp2p')
+const PeerId = require('peer-id')
 
-// specify options
-const options = {}
+;(async () => {
+  const peerId = await PeerId.create();
 
-// create libp2p
-const libp2p = new Libp2p(options)
+  // specify options
+  // peerId is required when Libp2p is instantiated via the constructor
+  const options = {
+    peerId
```

Comment:
Perhaps we should add to this example and the one above the modules as it is a mandatory property in the options. Can you add it like:

```js
const TCP = require('libp2p-tcp')
const MPLEX = require('libp2p-mplex')
const { NOISE } = require('libp2p-noise')
// ....
modules: {
    transport: [TCP],
    streamMuxer: [MPLEX],
    connEncryption: [NOISE]
}
```